pod.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: scheduler-pod-healthcheck
+spec:
+  containers:
+  - name: pause
+    image: gcr.io/google_containers/pause:0.8.0
```

Comment:
not sure where you got this default from, 3.0 is the latest https://console.cloud.google.com/gcr/images/google-containers/GLOBAL/pause

need to be able to override this somehow though. maybe some sed or envsubst magic in the script

Reply:
ta, think it was a random old issue + wasn't sure how you navigate gcr so didn't delve further into latest tag.
Do we need to override the tag or the image?

Reply:
I'd guess the whole image string including the tag